.github/workflows/test-push.yml

```
@@ -31,12 +31,12 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install flake8
-          pip install -r network-config-analyzer/requirements.txt
+          pip install network-config-analyzer
```

```suggestion
          pip install -r network-config-analyzer/requirements.txt
```
We still want to use `requirements.txt` in case `baseline-rules-verifier` will have dependencies of its own.